docs/source/commands.rst

```diff
@@ -0,0 +1,58 @@
+Command reference
+-----------------
+
+
+
+
```

Comment:
Nitpick: seems like 2 extra empty lines are here, compared to what we have elsewhere.

Reply:
:heavy_check_mark: